demos/super_resolution_demo/main.cpp

```diff
@@ -129,7 +155,7 @@ int main(int argc, char *argv[]) {
         /** Collect images**/
         std::vector<cv::Mat> inputImages;
         for (const auto &i : imageNames) {
-            cv::Mat img = cv::imread(i);
+            cv::Mat img = cv::imread(i, -1);
```

Comment:
It is clearer to use enum instead of its value.

Reply:
Replaced to `cv::IMREAD_UNCHANGED`